### PR TITLE
add missing changelog entries

### PR DESCRIPTION
**Pull request by feross**

### CODE REVIEW

**Comment by dcposch** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,48 @@
...
+
+- Allow deselecting the entire torrent with `deselect()` to happen earlier
+
+## v0.95.5 - 2016-07-26
+
+- Fix support for FileList input to client.seed()
+
+## v0.95.4 - 2016-07-26
+
+- Skip blocklist logic when opts.blocklist is not set
```
Comment (dcposch) [inline CHANGELOG.md]: What is that character?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,48 @@
...
+- Warn when WebTorrent is installed on Node.js older than v4.0.0.
+
+## v0.96.3 - 2016-08-22
+
+- Better docs for .renderTo()
+
+## v0.96.2 - 2016-08-20
+
+- replace 'hat' with 'randombytes'
+- better debug logs
```
Comment (dcposch) [inline CHANGELOG.md]: Uppercase

File: CHANGELOG.md (hunk 1 above)
Comment (feross) [inline CHANGELOG.md]: Which character?

File: CHANGELOG.md (hunk 2 above)
Comment (feross) [inline CHANGELOG.md]: Will fix

**Pull request merged**